workflows/diploshic_params.yaml

```diff
@@ -0,0 +1,18 @@
+#some parameters that should probably be moved to a .yaml file
+trainSampleNumber: 10 #the number of simulation replicates we want to generate for each file in our training set
+testSampleNumber: 10 #the number of simulations to create for each file in the test set
+sampleSize: 10 #the number of individuals in our population sample
+numSites: 55000 #total number of sites in our simulated window (i.e. S/HIC's subwindow size * 11)
```

Comment:
So in practice, we'd slide subwindows of 55000 bp over each 2Mb window simulated by stdpopsim? Or, just apply this to the center of the 2Mb window (e.g. where the simulated sweeps always are).

Do we need to think about increasing `numSites` for human-ish recombination/mutation rates?

Reply:
i think we should do a sliding thing? open to just applying this to one window too I guess.

55kb is meaningless other than the fact that `discoal` will be limited in how big region it can effectively simulate, but we could go bigger with human parameters I think